### DISCUSSION TITLE
Add option to delete a config entry while it is in reauth flow

### DISCUSSION THREAD

**Discussion by raman325** [home-assistant/frontend#8713] [Configuration]
Today I noticed that one of my config entries needed to be reconfigured because my login no longer worked. I wanted to get rid of the config entry, but there's no way to do that in the UI without completing the reauth flow first and then deleting it. I think you should be able to delete the entry without having to reauth first.

**Comment by frenck** [comment]
Oh nice idea 👍  I like that
